src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -965,17 +965,17 @@ void GatherForPredefinedReport(EnergyPlusData &state)
                 }
 
                 // Report table for every shading control state
-                const auto totalStates{Surface(iSurf).windowShadingControlList.size()};
+                const unsigned int totalStates = Surface(iSurf).windowShadingControlList.size();
```

Comment:
size_t to unsigned_int, I would have thought that would be size_t, but then I see it's use:

    const unsigned int totalStates = Surface(iSurf).windowShadingControlList.size();
    for (unsigned int i = 0; i < totalStates; ++i) {
        const int stateConstrNum = Surface(iSurf).shadedConstructionList[i];